src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -33,28 +33,58 @@ public BookController(BookRepository bookRepository) {
         return bookRepository.findAll();
     }
 
+    /**
+     * This method consults book by author
+     *
+     * @param bookAuthor
+     * @return the book object
```

Comment:
Just as a suggestion, you can use {@link Book} to link the book class in the comment 😄

Reply:
changos, you're right